DFC.ServiceTaxonomy.PageLocation/Views/PageLocationPart.Edit.cshtml

```diff
@@ -82,5 +88,10 @@
                 $fullUrlInput.val(locationString + $urlNameInput.val());
             }
         }
+
+        if ($('h1:contains("New Job Category")').length) {
```

Comment:
It seems sub-optimal to directly couple the page location part to its specific use within job categories, but perhaps we can look at that another time.